.github/ISSUE_TEMPLATE/2-bug-report-browser.md

```diff
@@ -0,0 +1,71 @@
+---
+name: "\U0001F41EBug report (browser extension)"
+about: Report a bug in the webhint browser extension
+title: "[Bug] Bug description"
+labels: "area:browser-extension, type:bug"
+---
+<!--
+
+Hi there 👋
+
+Thanks for taking the time to make webhint better! Before
+opening a new issue please take a look at the troubleshoot
+guide:
+https://webhint.io/docs/user-guide/troubleshoot/summary/
+and make sure to search the existing ones (even closed!)
+as they might contain information about workarounds,
+resolution, or progress updates.
+
+-->
+
+# 🐛 Bug report
+
+## Description
+
+<!-- ✍️ What were you doing? How do we repro your issue? -->
+
+## Environment
+
+* __webhint version:__ <!-- ✍️ Menu > Add-ons / Extensions -->
+* __Browser version:__ <!-- ✍️ Menu > Help > About  -->
+* __Operating system:__ <!-- ✍️ -->
+* __URL for which webhint failed:__ <!-- ✍️ -->
+
+## `webhint` configuration
+
+<details>
+<summary>webhint’s configuration</summary>
+
+Categories:
+* [x] Accessibility
+* [x] Compatibility
+* [x] Performance
+* [x] Pitfalls
+* [x] PWA
+* [x] Security
+
+Your target browsers:
+* [x] Recommended settings
+* [ ] Custom: <!-- ✍️ -->
+
+Ignored resources:
+* [x] None
+* [ ] Different origin
+* [ ] Custom: <!-- ✍️ -->
+
+</details>
+
+## Debug output
+
+Please include the raw output generated by `webhint`.
+
+<details>
+<summary>webhint’s raw output</summary>
```

Comment:
If we go with the above, then this should be something like

```suggestion
<summary>webhint’s Error details</summary>
```

Reply:
Done!